scripts/update/Updater.php

```diff
@@ -64,6 +69,17 @@ public function update($initialVersion)
         }
 
         $this->skip('0.1.0', '0.6.3');
+        if ($this->isVersion('0.6.3')) {
+            // events
+            /** @var EventManager $eventManager */
+            $this->getServiceManager()->register(BatteryEventHandler::SERVICE_ID, new BatteryEventHandler());
+            $eventManager = $this->getServiceManager()->get(EventManager::SERVICE_ID);
+            $eventManager->attach(BatteryCreatedEvent::class, [BatteryEventHandler::SERVICE_ID, 'logBatteryCreateEntry']);
```

Comment:
`logBatteryCreateEntry`-there's no such a function on the `BatteryEventHandler`. I only see `logBatteryChangesEntry`.

as well as `logBatteryModifyEntry`, see 2 lines ↓ below ↓